neuro-cli/tests/e2e/test_e2e_storage.py

```diff
@@ -116,7 +116,7 @@ def test_copy_local_file_to_platform_directory_explicit(
 
     helper.mkdir("folder", parents=True)
     # Upload local file to existing directory
-    helper.run_cli(["storage", "cp", "-t", helper.tmpstorage + "/folder", srcfile])
+    helper.run_cli(["storage", "cp", "-t", helper.tmpstorage + "folder", srcfile])
```

Comment:
Would you consider converting `helper.tmpstorage` from `str` to `URL` and applying `str(arg)` for all `run_cli()` arguments?
I believe, it makes tests more robust; `helper.tmpstorage / "folder"` is unambigous.

Reply:
Good idea, thanks